src/components/calculator.js

```diff
@@ -1,34 +1,107 @@
-import React from 'react';
+/* eslint-disable react/button-has-type */
+import { PropTypes } from 'prop-types';
+import calculate from '../logic/calculate';
+
+let counter = 0;
+let obj;
+const ans = document.querySelector('.answer');
+
+const Calculator = () => {
+  obj = {
+    total: null,
+    next: null,
+    operation: 'null',
+  };
```

Comment:
- [ ] Kindly consider using `useState` instead of object to handle the state of the calculator operation to practice on it because this is one of the goals of this project.
>![image](https://user-images.githubusercontent.com/53402600/234786010-818e929d-b9a6-44af-a32e-94851b338c25.png)

Reply:
Applied the changes as per instructions.